src/NServiceBus.Metrics.PerformanceCounters/PerformanceCountersFeature.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.Features;
+
+class PerformanceCountersFeature : Feature
+{
+    public PerformanceCountersFeature()
+    {
+        Defaults(s =>
+        {
+            options = s.EnableMetrics();
+            s.SetDefault(UpdateIntervalKey, TimeSpan.FromSeconds(2));
+        });
+    }
+
+    protected override void Setup(FeatureConfigurationContext context)
+    {
+        var logicalAddress = context.Settings.LogicalAddress();
+
+        var legacyInstanceNameMap = new Dictionary<string, CounterInstanceName?>
```

Comment:
I could be convinced either way. On the one hand, the typos and random spacing and abbreviations grate on my nerves. On the other hand, is it really bad enough to justify all this code?

Plus, it appears this is the only place we use JSON.NET so we could remove the dependency (and the corresponding ilmerge).